subprojects/docs/src/docs/userguide/depMngmt.adoc

```diff
@@ -544,6 +547,18 @@ Now Gradle will look for your dependencies in the JCenter repository. _jcenter()
             </sample>
 ++++
 
+[[sub:maven_google]]
+==== Maven Google repository
+
+The Google repository hosts Android-specific artifacts including the Android SDK. __Note:__ The contents of the repository cannot be explored from the browser.
```

Comment:
I requested additional info about this. I can remove it for now though I am sure people will ask.